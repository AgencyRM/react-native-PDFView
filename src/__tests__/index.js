import React from 'react';
import renderer from 'react-test-renderer';

import PDFVIew from '../index';

jest.mock('../RNPDFView', () => 'RNPDFView');

describe('PDFView', () => {
  it('renders', () => {
    const tree = renderer.create(<PDFVIew resource="http://site.com/pdf" />);

    expect(tree).toMatchSnapshot();
  });
});
