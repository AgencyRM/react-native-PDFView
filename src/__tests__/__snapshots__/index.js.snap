// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PDFView renders 1`] = `
<RNPDFView
  fadeInDuration={0}
  onError={[Function]}
  onLoad={[Function]}
  resource="http://site.com/pdf"
  resourceType="url"
  textEncoding="utf-8"
/>
`;
